app/components/dashboard/run/left-panel/component.js

```diff
@@ -61,12 +61,7 @@ export default Component.extend(FullScreenMixin, {
     init() {
         this._super(...arguments);
         let state = this.get('internalState');
-        let shouldButtonsAppear = state.currentInstanceId;
-        if (shouldButtonsAppear) {
-            this.set('hasSelectedTaleInstance', true);
-        } else {
-            this.set('hasSelectedTaleInstance', false);
-        }
+        state.set('currentInstanceId', this.get('model').get('instance')._id);
```

Comment:
I am getting an error from this line when refreshing the "Run" view.

For example - upon navigating to https://dashboard.local.wholetale.org/run/5d6072a40b7426cac492ad17, the view never renders and I get an error in the console:
```
TypeError: Cannot read property '_id' of undefined
    at o.init (wholetale-e526907ced1132f1aa2e27112b55b6ed.js:80)
    at o.n [as init] (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:3148)
    at new o (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:3057)
    at Function.u.create (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:3066)
    at e.create (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:1708)
    at n.create (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:2060)
    at Object.evaluate (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:1257)
    at e.evaluate (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:1146)
    at e.evaluateSyscall (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:1387)
    at e.evaluateInner (vendor-d660d4a9eb3aed0fe0d41d8d4c22d229.js:1381)
```

Reply:
I think I've found a way to totally remove `currentInstanceId` from the dashboard, which should fix this.